### PR TITLE
[release-1.15] Remove ES performance tests and patches

### PR DESCRIPTION
**Pull request by skonto**
**What this PR does / why we need it**:

- Downstream QE uses InfluxDB.
- Tests against ES are not being used at the moment and block Konflux see [here](https://github.com/openshift-knative/serving/pull/912#issuecomment-2407425363)
- I will update main branch too.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -30,14 +30,6 @@ test-e2e:
 	./openshift/e2e-tests.sh
 .PHONY: test-e2e
 
-perf-tests:
```
Comment (skonto) [inline Makefile:33]: CI jobs will be updated after this is merged

**Review by dsimansk — APPROVED** [review]

**Comment by skonto** [comment]
@dsimansk maybe merge as the contract thing is failing in #912 too? Unless you want to override all the checks?

**Comment by ReToCode** [comment]
The enterprise-contract things fail on all Serving repos as we still use quay.io and other stuff that is "not allowed" in Konflux, so overriding for now is fine.

**Comment by ReToCode** [comment]
/override "Red Hat Konflux / serverless-operator-135-enterprise-contract / kn-serving-activator-115"
/override "Red Hat Konflux / serverless-operator-135-enterprise-contract / kn-serving-autoscaler-115"
/override "Red Hat Konflux / serverless-operator-135-enterprise-contract / kn-serving-autoscaler-hpa-115"
/override "Red Hat Konflux / serverless-operator-135-enterprise-contract / kn-serving-cleanup-115"
/override "Red Hat Konflux / serverless-operator-135-enterprise-contract / kn-serving-controller-115"
/override "Red Hat Konflux / serverless-operator-135-enterprise-contract / kn-serving-dataplane-probe-115"
/override "Red Hat Konflux / serverless-operator-135-enterprise-contract / kn-serving-default-domain-115"
/override "Red Hat Konflux / serverless-operator-135-enterprise-contract / kn-serving-load-test-115"
/override "Red Hat Konflux / serverless-operator-135-enterprise-contract / kn-serving-queue-115"
/override "Red Hat Konflux / serverless-operator-135-enterprise-contract / kn-serving-real-traffic-test-115"
/override "Red Hat Konflux / serverless-operator-135-enterprise-contract / kn-serving-reconciliation-delay-115"
/override "Red Hat Konflux / serverless-operator-135-enterprise-contract / kn-serving-rollout-probe-115"
/override "Red Hat Konflux / serverless-operator-135-enterprise-contract / kn-serving-scale-from-zero-115"
/override "Red Hat Konflux / serverless-operator-135-enterprise-contract / kn-serving-storage-version-migration-115"
/override "Red Hat Konflux / serverless-operator-135-enterprise-contract / kn-serving-test-autoscale-115"
/override "Red Hat Konflux / serverless-operator-135-enterprise-contract / kn-serving-test-failing-115"
/override "Red Hat Konflux / serverless-operator-135-enterprise-contract / kn-serving-test-grpc-ping-115"
/override "Red Hat Konflux / serverless-operator-135-enterprise-contract / kn-serving-test-hellohttp2-115"
/override "Red Hat Konflux / serverless-operator-135-enterprise-contract / kn-serving-test-hellovolume-115"
/override "Red Hat Konflux / serverless-operator-135-enterprise-contract / kn-serving-test-helloworld-115"
/override "Red Hat Konflux / serverless-operator-135-enterprise-contract / kn-serving-test-httpproxy-115"
/override "Red Hat Konflux / serverless-operator-135-enterprise-contract / kn-serving-test-pizzaplanetv1-115"
/override "Red Hat Konflux / serverless-operator-135-enterprise-contract / kn-serving-test-pizzaplanetv2-115"
/override "Red Hat Konflux / serverless-operator-135-enterprise-contract / kn-serving-test-readiness-115"
/override "Red Hat Konflux / serverless-operator-135-enterprise-contract / kn-serving-test-revisionfailure-115"
/override "Red Hat Konflux / serverless-operator-135-enterprise-contract / kn-serving-test-runtime-115"
/override "Red Hat Konflux / serverless-operator-135-enterprise-contract / kn-serving-test-servingcontainer-115"
/override "Red Hat Konflux / serverless-operator-135-enterprise-contract / kn-serving-test-sidecarcontainer-115"
/override "Red Hat Konflux / serverless-operator-135-enterprise-contract / kn-serving-test-singlethreaded-115"
/override "Red Hat Konflux / serverless-operator-135-enterprise-contract / kn-serving-test-slowstart-115"
/override "Red Hat Konflux / serverless-operator-135-enterprise-contract / kn-serving-test-timeout-115"
/override "Red Hat Konflux / serverless-operator-135-enterprise-contract / kn-serving-test-volumes-115"
/override "Red Hat Konflux / serverless-operator-135-enterprise-contract / kn-serving-test-wsserver-115"
/override "Red Hat Konflux / serverless-operator-135-enterprise-contract / kn-serving-webhook-115"
/override "Red Hat Konflux / serverless-operator-135-enterprise-contract / pr group remove_perf_tests"

**Pull request merged**